torch_points3d/modules/SparseConv3d/nn/torchsparse.py

```diff
@@ -41,7 +41,7 @@ def __init__(
             stride=stride,
             dilation=dilation,
             bias=bias,
-            transpose=True,
+            transposed=True,
```

Comment:
which version of torchsparse do you use ?

Reply:
latest (1.4)